examples/long_running.py

```diff
@@ -0,0 +1,117 @@
+"""Example specification for a consumption-savings model with health and leisure."""
+
+import jax.numpy as jnp
+
+# ======================================================================================
+# Numerical parameters and constants
+# ======================================================================================
+N_GRID_POINTS = {
+    "wealth": 100,
+    "health": 100,
+    "consumption": 100,
+    "exercise": 200,
+}
+
+RETIREMENT_AGE = 65
+
+# ======================================================================================
+# Model functions
+# ======================================================================================
+
+
+# --------------------------------------------------------------------------------------
+# Utility function
+# --------------------------------------------------------------------------------------
+def utility(consumption, working, health, exercise, disutility_of_work):
+    return jnp.log(consumption) - (disutility_of_work - health) * working - exercise
+
+
+# --------------------------------------------------------------------------------------
+# Auxiliary variables
+# --------------------------------------------------------------------------------------
+def labor_income(wage, working):
+    return wage * working
+
+
+def working(leisure):
```

Comment:
```suggestion
def leisure(working):
```
`working` just needs to be the choice to fit my brain.